### PR TITLE
Cluster-autoscaler: other autoscalers in FAQ

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @MaciekPytel @fgrzadkowski

### CODE REVIEW

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ underutilized or completely empty and then CA will delete such unneded nodes.
 * Specify requests for your pods.
 * Use PodDisruptionBudgets to prevent pods from being deleted (if needed).
 * Check if your cloud provider quota is big enough before specifying min/max settings for your node pools.
+* Do not run any additional node group autoscalers (especially these from your cloud provider).
+
+### Is Cluster Autoscaler compatible with CPU-based autoscalers.
```
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: Maybe change autoscalers to "node autoscalers"? HPA is a CPU-based autoscaler that is meant to be used with CA.

File: cluster-autoscaler/FAQ.md [hunk 2]
```diff
@@ -36,6 +36,13 @@ underutilized or completely empty and then CA will delete such unneded nodes.
 * Specify requests for your pods.
 * Use PodDisruptionBudgets to prevent pods from being deleted (if needed).
 * Check if your cloud provider quota is big enough before specifying min/max settings for your node pools.
+* Do not run any additional node group autoscalers (especially these from your cloud provider).
```
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: I would change these to those

File: cluster-autoscaler/FAQ.md (hunk 1 above)
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: Added. However node/cluster is in the line below.

Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: Agreed, but the question is in rendered with larger, bold font and it's easy to just notice it and the NO in caps without reading the rest.

File: cluster-autoscaler/FAQ.md (hunk 2 above)
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: done.

File: cluster-autoscaler/FAQ.md (hunk 1 above)
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: done.

**Pull request merged**